doc/k8s/README.md

```diff
@@ -0,0 +1,53 @@
+# Running SQLFlow on Kubernetes
```

Comment:
Can put in directory `doc` and rename to `run_on_kubernetes.md`?

Reply:
done, but maybe we also need the `k8s` directory so that we can put the yaml files in it ?